jobbergate-cli/jobbergate_cli/subapps/applications/app.py

```diff
@@ -314,3 +320,37 @@ def delete(
         """,
         subject="Application delete succeeded",
     )
+
+
+@app.command()
+@handle_abort
+def download_files(
+    ctx: typer.Context,
+    id: Optional[int] = typer.Option(
+        None,
+        help=f"The specific id of the application. {ID_NOTE}",
+    ),
+    identifier: Optional[str] = typer.Option(
+        None,
+        help=f"The human-friendly identifier of the application. {IDENTIFIER_NOTE}",
+    ),
+):
+    """
+    Download the files from an application to the current working directory.
+    """
+    jg_ctx: JobbergateContext = ctx.obj
+    result = fetch_application_data(jg_ctx, id=id, identifier=identifier)
+    saved_files = save_application_files(
+        application_data=result,
+        destination_path=pathlib.Path.cwd(),
```

Comment:
shouldn't this path be an input from the user?

Reply:
That could be done, but the current working directory was specified in the requirements.